alpha/power.h

```diff
@@ -1,7 +1,9 @@
 #ifndef __TS_POWER__
 #define __TS_POWER__
 
-#include "src\FunctionFsm\src\FunctionFSM.h"
+#ifndef FunctionFsm
+  #include "src\FunctionFsm\src\FunctionFSM.h"
```

Comment:
doh, #pragma once is probably not supported in FunctionFSM.h ? why do we need to do this?